CONTRIBUTING.md

```diff
@@ -39,3 +39,72 @@ Apply Linter checks with the following instructions
 ## Contributing to Issue Finder
 
 - We welcome all meaningful contributions to Issue Finder. If you see any areas of improvement or wish to contribute to existing issues, please connect with our maintainers directly on the issue, and we will be pleased to help you as much as possible.
+
+
+## Contributing - Open Source
+
+**1.** Fork the repository.
+
+**2.** Clone your forked copy of the project into your local system.
+
+```
+git clone --depth 1 https://github.com/yourusername/issue-finder.git
+```
+
+**3.** Navigate to the project directory. 📁
+
+```
+cd healthphilics
```

Comment:
Thank you for contributing to Issue Finder!

Please change this to `cd issue-finder`.